packages/murmur2/test/cli_murmur2.ml

```diff
@@ -1,8 +1,24 @@
-let () =
-  if Array.length Sys.argv <> 2 then (
-    Printf.eprintf "Usage: %s <input_string>\n" Sys.argv.(0);
-    exit 1);
-
+if Array.length Sys.argv <> 2 then (
+  Printf.eprintf "Usage: %s <input_string>\n" Sys.argv.(0);
+  exit 1)
+else if Sys.argv.(1) = "--mal-formed" then (
```

Comment:
Is this flag used?

Reply:
Nop, it was a previous attempt to create the tests for malformed, forgot to delete it.